Documentation/network/concepts/masquerading.rst

```diff
@@ -142,7 +139,7 @@ configure the ``ip-masq-agent`` as above.
 
 .. note::
 
-    eBPF based masquerading is currently not supported for IPv6 traffic.
+    The eBPF-based ip-masq-agent is currently not supported for IPv6 traffic.
```

Comment:
I'd like to see "currently" removed, but I'm willing to defer it as out of scope for the PR.

Reply:
So are we not going to see IPv6 support for ip-masq-agent? Or why do you want to remove "currently"?

Reply:
My understanding from Sarah's feedback is that having “currently” in the docs is not a good practice - you never really when “currently” refers to (it refers to the date of writing obviously, but when reading the docs, how do you figure this one out?).

Sarah, I'll remove that word for the next iteration.

As for the ip-masq-agent with IPv6, it is coming [as a follow-up PR](https://github.com/cilium/cilium/pull/23219), it's _currently_ waiting for this one to be merged.

Reply:
Now updated in the docs.